### ISSUE TITLE
Fix CSV file generation so it can be uploaded directly to moodle.

### ISSUE THREAD

**Issue by ludvi025** [ludvi025/graid#11]
This has something to do with newline characters. The script uses csv.DictWriter, which requests that files be opened with the `newline=''` parameter. Probably has something to do with this.

**Comment by tompostler** [comment]
I don't think it has anything to do with the `newline=''` parameter as that just handles the writing of the file.

However, I did find that the current method of uploading the CSV to Google Drive and then downloading it again makes a few interesting (and hopefully useful to know) changes:
- There is no newline at the end of the file from Drive
- The newlines from Drive are `\n` instead of `\r\n`

Also, it would help to know if this is for all types of CSVs, or just ones where a student has multiple names.

**Comment by tompostler** [comment]
This _might_ fix it (an optional parameter to the CSV writer):

```
quoting=csv.QUOTE_NONNUMERIC
```

_a la https://github.com/manly-man/moodle-destroyer-tools_
